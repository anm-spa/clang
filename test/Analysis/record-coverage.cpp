// RUN: %clang_cc1 -analyze -analyzer-checker=core -analyzer-config record-coverage=%T %s
// RUN: FileCheck -input-file %T/%s.gcov %s.expected

int main() {
  int i = 2;
  ++i;
  if (i != 0) {
    ++i;
  } else {
    --i;
  }
}
