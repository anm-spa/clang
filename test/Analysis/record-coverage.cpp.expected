// CHECK:      -:4:int main() {
// CHECK-NEXT: 1:5:  int i = 2;
// CHECK-NEXT: 1:6:  ++i;
// CHECK-NEXT: 1:7:  if (i != 0) {
// CHECK-NEXT: 1:8:    ++i;
// CHECK-NEXT: -:9:  } else {
// CHECK-NEXT: #####:10:    --i;
// CHECK-NEXT: -:11:  }
// CHECK-NEXT: -:12:}
